datadog_checks_dev/datadog_checks/dev/tooling/commands/validate/metadata.py

```diff
@@ -182,16 +182,28 @@
     'exacore',
```

Comment:
Ideally this should be split into 3 PRs:
- ddev change
- snmp snmp/metadata.csv change
- new tile

It makes it easier to review and easier to revert if needed.

Reply:
ok, will split it